drivers/base/bootvid/i386/pc/vga.c

```diff
@@ -98,17 +98,9 @@ do {                                                        \
     WRITE_REGISTER_UCHAR((_PixelPtr), (UCHAR)(_TextColor)); \
 } while (0);
 
-#ifdef CHAR_GEN_UPSIDE_DOWN
-# define GetFontPtr(_Char) &FontData[_Char * BOOTCHAR_HEIGHT] + BOOTCHAR_HEIGHT - 1;
-# define FONT_PTR_DELTA (-1)
-#else
-# define GetFontPtr(_Char) &FontData[_Char * BOOTCHAR_HEIGHT];
-# define FONT_PTR_DELTA (1)
-#endif
-
 VOID
 NTAPI
-DisplayCharacter(
+VidpDisplayCharacter(
```

Comment:
```suggestion
DisplayCharacter(
```
to emphasise it's a per-arch dependent global helper function and not a generic private bootvid helper.